packages/components/psammead-headings/src/index.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Headline, SubHeading } from './index';
+
+storiesOf('Headline', module).add('default', () => (
+  <Headline>This is my headline.</Headline>
+));
+
+storiesOf('SubHeading', module).add('default', () => (
+  <SubHeading text="This is a SubHeading">This is a SubHeading</SubHeading>
```

Comment:
I think this can be refactored but in a later PR. Have raised https://github.com/BBC-News/psammead/issues/88